docs/studio/cli/README.md

```diff
@@ -46,6 +47,21 @@ studio <command> <arguments> [flags]
 
 You can see all of these commands available by either running the `help` command or passing the help flag `-h, --help`.
 
+### Deployment
```

Comment:
note: the Studio CLI docs changes start here—should be accurate afaik (i tested each in prod). it adds/updates the `deployment`, `import-data`, and `project` commands, and i fixed incorrect descriptions i wrote for `login` command flags.